lib/node_modules/@stdlib/stats/base/dists/bernoulli/pmf/src/main.c

```diff
@@ -0,0 +1,49 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/bernoulli/pmf.h"
+#include "stdlib/math/base/assert/is_nan.h"
+
+/**
+* Evaluates the probability mass function (PMF) for a Bernoulli distribution with success probability `0 <= p <= 1`.
+*
+* @param x    input value
+* @param p    success probability
+* @return     evaluated PMF
+*
+* @example
+* double y = stdlib_base_dists_bernoulli_pmf( 0.5, 0.5 );
+* // returns 0.5
+*/
```

Comment:
This example has to be incorrect, as the PMF can only have non-zero values for `0` and `1`.